search/search-client-elasticsearch/src/main/java/io/camunda/search/clients/ElasticsearchSearchClient.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.elasticsearch._types.ElasticsearchException;
+import co.elastic.clients.elasticsearch.core.SearchRequest;
+import co.elastic.clients.elasticsearch.core.SearchResponse;
+import io.camunda.search.clients.core.SearchQueryRequest;
+import io.camunda.search.clients.core.SearchQueryResponse;
+import io.camunda.search.transformers.ElasticsearchTransformers;
+import io.camunda.search.transformers.SearchTransfomer;
+import io.camunda.search.transformers.search.SearchResponseTransformer;
+import io.camunda.zeebe.util.Either;
+import java.io.IOException;
+
+public final class ElasticsearchSearchClient implements CamundaSearchClient {
+
+  private final ElasticsearchClient client;
+  private final ElasticsearchTransformers transformers;
+
+  public ElasticsearchSearchClient(final ElasticsearchClient client) {
+    this(client, new ElasticsearchTransformers());
+  }
+
+  public ElasticsearchSearchClient(
+      final ElasticsearchClient client, final ElasticsearchTransformers transformer) {
+    this.client = client;
+    this.transformers = new ElasticsearchTransformers();
```

Comment:
Should it be `this.transformers = transformer`? Or it is the case that we are creating a new transformer here? (So it would be not necessary the second parameter)

Reply:
Copy paste error, it should be `this.transformers = transformer`